src/components/APIDocsCard.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+import {
+  Card,
+  CardHeader,
+  CardBody,
+  CardFooter,
+  IconContentCopy,
+  Link,
+  CardGroup,
+} from "@trussworks/react-uswds";
+
+export const APIDocsCard = () => {
+  return (
+    <CardGroup>
+      <Card>
+        <CardHeader>
+          <h2>API Documentation</h2>
+        </CardHeader>
+
+        <CardBody>
+          <IconContentCopy size={4} aria-label={"Icon for API Docs card"} />
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+        </CardBody>
+
+        <CardFooter>
+          <Link href={"https://www.epa.gov/"}>Documentation</Link>
```

Comment:
I think this should be a react-router-dom <Link> and we should link to the pages that don't exist yet instead of the epa.gov sites. 

https://v5.reactrouter.com/web/api/Link
https://v5.reactrouter.com/web/guides/quick-start